docs/ADDING_MACHINES.md

```diff
@@ -78,14 +78,26 @@ MIMachineEvents.registerMachines(event => {
       items => items.addSlots(42, 27, 3, 3).addSlots(139, 27, 1, 3), fluids => {},
       /* MODEL CONFIGURATION */
       // front overlay?, top overlay?, side overlay?
-      true, true, false,
```

Comment:
It's good practice to leave an extra comma (in the languages that allow it). :wink: